wagtailsharing/tests/test_views.py

```diff
@@ -15,6 +15,10 @@
 from wagtailsharing.views import ServeView
 
 
+def before_route_page_hook_returns_http_response(view, request, path):
```

Comment:
Maybe it's worth renaming the existing `before_hook_returns_http_response` to something generic like `hook_returns_http_response` and reusing it in multiple tests instead of duplicating?

Reply:
Unfortunately, the hooks have different signatures, based on what's available in their call context.

Reply:
Done!